letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -68,10 +68,12 @@ for arg in "$@" ; do
       NO_BOOTSTRAP=1;;
     --help)
       HELP=1;;
-    --noninteractive|--non-interactive|renew)
-      ASSUME_YES=1;;
+    --noninteractive|--non-interactive)
+      NONINTERACTIVE=1;;
```

Comment:
If we're not actually using this flag for anything, we shouldn't set it.

Reply:
And if we are leaving this, we should probably also check the short-form.

Reply:
`NONINTERACTIVE` is the flag we use to determine it's OK to rebootstrap without user intervention. See https://github.com/certbot/certbot/blob/v0.20.0/certbot-auto#L833.

And we do check the short form but due to the fun of POSIX compatible shell scripting, we have to check for the short form differently. See https://github.com/certbot/certbot/blob/v0.20.0/certbot-auto#L83.